widget/playground/src/containers/StyleLayout/StyleLayout.styles.ts

```diff
@@ -34,17 +34,43 @@ export const Tabs = styled('div', {
   display: 'flex',
   border: '3px solid $neutral100',
   flexDirection: 'row',
+  position: 'relative',
 });
 
 export const Tab = styled(Button, {
   color: '$neutral700',
   backgroundColor: 'transparent',
-  '&:hover': {
-    backgroundColor: '$info100',
-    color: '$secondary500',
+  zIndex: 10,
+  variants: {
+    isActive: {
+      true: {
+        $$color: '$colors$background',
+        [`.${darkTheme} &`]: {
+          color: '$colors$foreground',
+        },
+        color: '$$color',
+      },
+      false: {
+        '&:hover': {
+          backgroundColor: '$info100',
```

Comment:
please double-check this hover in dark theme.

Reply:
fixed.
(Note: I think in Figma, we do not have enough information about the dark playground theme in different sections.)